### ISSUE TITLE
Scoreboard bug

### ISSUE THREAD

**Issue by ChimpGamer** [MysticCity/MinigamesAPI#260]
I have problems with the scoreboard.
EDIT:
Spaces don't work either.

  custom_scoreboard:
    line0: '&5Map:'
    line1: <arena>
    line2: '&f'
    line3: '&5Spelers:'
    line4: <playercount>/<maxplayercount>
    line5: '&f'
    line6: '&6www.torchcraft.nl'
  custom_lobby_scoreboard:
    line0: '&5Map:'
    line1: <arena>
    line2: '&f'
    line3: '&5Spelers:'
    line4: <playercount>/<maxplayercount>
    line5: '&f'
    line6: '&6www.torchcraft.nl'

[http://i.imgur.com/DP2DqcF.png](url)

**Comment by mepeisen** [comment]
First of all the placeholder "<arena> is not implemented. I will add it within next release.

The other placeholders are currently only working in the way the default configuration stores them.
for Example: "Players:<playercount>"

In 2.0 there may be some more options for making individual scoreboards.


The colon (':') is currently eaten because it has a special meaning for minigameslib.

I will check the line ordering.

Anything I missed?

**Comment by ChimpGamer** [comment]
Nope, but maybe an option to allow the use of a FeatherBoard Scoreboard is better. So you can set the name of the featherboard scoreboard in the config and Featherboard will be able to use placeholders like the arena placeholder etc. (Only if possible)

Thanks!

**Comment by mepeisen** [comment]
I am already planning to implement a bridge to placeholder api. This is done via an additional plugin, not within the core or minigames-API. Other plugins may be supported too. Please append a link to the plugin at issue #192

**Comment by mepeisen** [comment]
moved to 2.0
